packages/core/release-content.txt

```diff
@@ -0,0 +1,140 @@
+package
```

Comment:
Where do you generate these files ?

Reply:
just manually, by running the same command as in the `check-release-content.sh`